h2olog

```diff
@@ -814,19 +814,19 @@ if len(sys.argv) > 1 and sys.argv[1] == "quic":
 
 try:
     h2o_pid = 0
-    allowed_quic_event = None
-    allowed_res_header_name = None
+    allowed_quic_event = set()
+    allowed_res_header_name = set()
     verbose = False
     opts, args = getopt.getopt(sys.argv[optidx:], 'vp:t:s:')
     for opt, arg in opts:
         if opt == "-p":
             h2o_pid = arg
         elif opt == "-t":
-            allowed_quic_event = arg
+            allowed_quic_event.update(arg.replace(":", "__").split(","))
```

Comment:
This also includes a  fix to replace ":" with "__", to accept fully-specified names instead of raw event names.